lib/MusicBrainz/Server/Entity/URL/Sidebar.pm

```diff
@@ -86,7 +86,8 @@ use constant FAVICON_CLASSES => {
     'indiegogo.com'             => 'indiegogo',
     'kickstarter.com'           => 'kickstarter',
     'setlist.fm'                => 'setlistfm',
-    'vimeo.com'                 => 'vimeo'
+    'vimeo.com'                 => 'vimeo',
+    'songkick.com'              => 'songkick'
```

Comment:
Just a note, you can add a trailing comma after the last element in the list, and it'll make additions cleaner in the future (note how it has to remove the vimeo line first to append a comma).